pgxn/neon/file_cache.c

```diff
@@ -223,7 +223,7 @@ lfc_change_limit_hook(int newval, void *extra)
 	 * Stats collector detach shared memory, so we should not try to access shared memory here.
 	 * Parallel workers first assign default value (0), so not perform truncation in parallel workers.
 	 */
-	if (!lfc_ctl || !UsedShmemSegAddr || IsParallelWorker())
+	if (!lfc_ctl || !MyProc || !UsedShmemSegAddr || IsParallelWorker())
```

Comment:
It would be great to add to the above comment about why not having a PGPROC structure is an issue. Perhaps also mention it in the commit message.

Reply:
Done

Reply:
Thanks. Cleaning it up a bit...

```
The Postmaster can handle SIGHUP and it has access to shared memory (UsedShmemSegAddr != NULLL), but is has no PGPROC.
```

Reply:
Should we also add something to the GUC description about "This has no affect in the Postmaster."

Reply:
> Should we also add something to the GUC description about "This has no affect in the Postmaster."
I do not think so. I even wonder why configuration changes are ever processed by postmaster...